package.json

```diff
@@ -57,6 +59,7 @@
     "react": "^16.8.5",
     "react-dom": "^16.8.5",
     "react-final-form": "^6.3.0",
+    "react-intl": "^2.9.0",
```

Comment:
Why react-intl v2? Could this be v5?

Reply:
it will be before this is moved out of draft. The terra-application integration branch hasn't been updated to v5 yet.